src/radiation/radiation_system.hpp

```diff
@@ -1447,7 +1447,7 @@ void RadSystem<problem_t>::AddSourceTerms(array_t &consVar, arrayconst_t &radEne
 				quokka::valarray<double, nGroups_> F_D{};
 
 				const double resid_tol = 1.0e-11; // 1.0e-15;
-				const int maxIter = 50;
+				const int maxIter = 100;
```

Comment:
What is the motivation for changing this?

Reply:
With the dust-gas coupling model, it usually takes more iterations to converge, so I have to increase maxIter.